lib/AST/ASTImporter.cpp

```diff
@@ -6927,14 +6929,8 @@ Decl *ASTImporter::Import(Decl *FromD) {
   ToD = Importer.Visit(FromD);
   if (!ToD)
     return nullptr;
+  Imported(FromD, ToD);
```

Comment:
I wonder if this Imported call is redundant in most of the cases since Visit functions actually call this after a declaration is imported.